storj/uploader.py

```diff
@@ -396,13 +379,10 @@ def file_upload(self):
 
         success = False
         try:
-            # TODO
-
-            # This is the actual upload_file method
+            # Post an upload_file request
             response = self.client._request(
```

Comment:
yes, what i wanted to do is providing a working uploader, then we can edit and improve it, if there is some duplicated code